docs/articles/documentation/continuous-integration/bitbucket-pipelines.md

```diff
@@ -0,0 +1,112 @@
+
+### Option 1 - Use TestCafe Docker Image
+
+Specify the `testcafe/testcafe` image name in the `image` field of the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html). Then, configure a pipeline that installs the project's dependencies and triggers a custom TestCafe launcher when you push commits and create pull requests.
+
+```yaml
+image: testcafe/testcafe
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          script:
+            - npm ci
+            - /opt/testcafe/docker/testcafe-docker.sh firefox:headless,chromium tests/**/*
+
+  branches:
+    master:
+      - step:
+          script:
+            - npm ci
+            - /opt/testcafe/docker/testcafe-docker.sh firefox:headless,chromium tests/**/*
+```
+
+The custom launcher script `/opt/testcafe/docker/testcafe-docker.sh` prepares the environment to run a browser and starts TestCafe. Its arguments are standard TestCafe [command line parameters](../using-testcafe/command-line-interface.md).
+
+Commit and push this file to your repository.
+
+### Option 2 - Install TestCafe on a Docker Image
+
+Use the `image` field in the [bitbucket-pipelines.yml file](https://confluence.atlassian.com/bitbucket/configure-bitbucket-pipelines-yml-792298910.html) to specify a desired Docker image with Node.js and browsers installed. Then, configure a pipeline that installs the project's dependencies and runs tests when you push commits and create pull requests.
+
+```yaml
+# Replace '10.14' with the latest Node.js LTS version
+# available on Docker Hub
+image: circleci/node:10.14-browsers
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          script:
+            - npm ci
+            - npm test
+
+  branches:
+    master:
+      - step:
+          script:
+            - npm ci
+            - npm test
+```
+
+To specify how npm should run TestCafe, add the `test` script to the project's `package.json` file and execute the `testcafe` command in this script.
+
+The following example shows a command that runs tests in Chromium in headless mode.
+
+```json
+"scripts": {
+    "test":  "testcafe 'chromium:headless --no-sandbox --disable-setuid-sandbox --window-size=1920x1080' tests/index-test.js"
+}
+```
+
+For more information on how to configure a test run with the `testcafe` command, see [Command Line Interface](../using-testcafe/command-line-interface.md).
+
+Commit the changes and push them to your repository.
+
+### Tip: Start a Custom Web Server
+
+If you need to start a custom Web server to host your application, use the [--app](../using-testcafe/command-line-interface.md#-a-command---app-command) TestCafe option followed by a command that starts this server.
+TestCafe executes this command automatically before tests are launched. After tests finish, TestCafe stops the server.
```

Comment:
"TestCafe executes this command automatically before tests are launched." -> TestCafe executes this command before tests are launched.